pkg/api/restore.go

```diff
@@ -47,23 +47,38 @@ const (
 	KATA_TAR_FILE_RECEIVER_PORT  = 9998
 )
 
-func (s *service) setupStreamerServe(dumpdir string) (*exec.Cmd, error) {
+func (s *service) setupStreamerServe(dumpdir string, num_pipes int32) {
 	buf := new(bytes.Buffer)
-	cmd := exec.Command("cedana-image-streamer", "--dir", dumpdir, "serve")
+	out := new(bytes.Buffer)
+	cmd := exec.Command("sudo", "cedana-image-streamer", "--dir", dumpdir, "--num-pipes", fmt.Sprint(num_pipes), "serve")
 	cmd.Stderr = buf
+	cmd.Stdout = out
+	logpath := "/var/log/cedana-image-streamer-restore.log"
```

Comment:
also same as before comment, we should try and get these logs into the zerolog logger

Reply:
so for each one (cmd.Stderr, cmd.Stdout), it might look like a gofunc for each, where you're looping through each one and logging it. Something like: 
```go
pipe := bufio.NewReader(stdout)
 for {
            line, err := pipe.ReadString('\n')
            if err != nil {
                break
            }
            log.Info().Msg(line)
```

and stdout is: 
```go 
stdout, err := cmd.StdoutPipe()
    if err != nil {
        return err
    }
```